app/src/main/res/layout-sw600dp-land/onboarding_slide_final.xml

```diff
@@ -16,67 +16,59 @@
 
     <ImageView
       android:id="@+id/slide_image_view"
-      android:layout_width="0dp"
+      android:layout_width="wrap_content"
       android:layout_height="0dp"
-      android:contentDescription="@string/onboarding_slide_3_title"
       android:adjustViewBounds="true"
+      android:contentDescription="@string/onboarding_slide_3_title"
       android:scaleType="fitXY"
-      android:src="@drawable/ic_landscape_onboarding_3"
+      android:src="@drawable/ic_landscape_onboarding_3_tablet"
       app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintDimensionRatio="H, 10:9"
```

Comment:
Ditto as above.

Reply:
@MohamedMedhat1998 Please make sure that you reply to each comment. Thanks.

Reply:
Ok  
Followed the same approach
> 
> 
> 40:39 is perfect 👌
> ![screenshot-2020-05-31_14 34 29 311](https://user-images.githubusercontent.com/8533363/83352492-0b42f180-a34c-11ea-98db-99d2794fed8e.png)
> ![screenshot-2020-05-31_14 35 16 889](https://user-images.githubusercontent.com/8533363/83352498-1433c300-a34c-11ea-9ec7-af541fd50704.png)
> ![screenshot-2020-05-31_14 35 40 305](https://user-images.githubusercontent.com/8533363/83352501-185fe080-a34c-11ea-93e8-c7fbe909c167.png)
> ![screenshot-2020-05-31_14 35 00 609](https://user-images.githubusercontent.com/8533363/83352496-10a03c00-a34c-11ea-9853-b5e593f0cd0e.png)

Reply:
Great. Thanks.